.github/actions/amplify-preview/action.yaml

```diff
@@ -0,0 +1,141 @@
+name: Amplify Preview
```

Comment:
Can you move this per [this RFD](https://github.com/gravitational/shared-workflows/blob/main/rfd/0001-tooling-requirements.md#repo-structure) and add a README?